tpm/aziot-tpm-common-http/src/lib.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+//! Request/Response types used by the TPM Service's HTTP API.
+
+#![deny(rust_2018_idioms)]
+#![warn(clippy::all, clippy::pedantic)]
+#![allow(clippy::wildcard_imports)] // to use `super::*`
```

Comment:
But the `use super::*;` isn't doing anything in the first place?

Reply:
It imports `Serialize` and `Deserialize` into scope.

This is a stylistic choice, since I'm not a fan of using qualified paths for derive macros (especially those as foundational as `serde`), and would rather not `use serde::{Deserialize, Serialize}` in every sub-block.